package.json

```diff
@@ -18,7 +18,7 @@
         "compile-scripts": "tsc -p scripts",
         "not-needed": "node scripts/not-needed.js",
         "update-codeowners": "node scripts/update-codeowners.js",
-        "test": "node node_modules/types-publisher/bin/tester/test.js --run-from-definitely-typed",
+        "test": "node --max_old_space_size=4096 node_modules/types-publisher/bin/tester/test.js --run-from-definitely-typed",
```

Comment:
Be careful with such changes. This gives 4GB to each node process but the travis CI machines most likely don't have that much. A while ago I reduced the number of parallel processes as the sum of all processes causes OOM.

Reply:
I just made a [change](https://github.com/Microsoft/types-publisher/pull/607) to make this unnecessary and will be reverting this change.